appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
+  skip_symbols: false
+  artifact: core 
+  on:
+      appveyor_repo_tag: true
```

Comment:
Now have appveyor updating build version from tags, yeay!

Please see: fd8de4d8147915d1ffe33dcc3b4bd388fa6003a5

After creating this: https://github.com/cryosharp/fortress/releases/tag/0.5.11

Had the following build run: https://ci.appveyor.com/project/fir3pho3nixx/fortress/build/0.5.11

Going to move on to securing NuGet with env var.